### DISCUSSION TITLE
Estimated Amounts for Charge Lines, no CPT/HCPCS

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#132] [Q&A] [answered]
Hi There All - I have a question around Estimated Amounts for Charge Lines that have no CPT/HCPCS OR where the CPT/ HCPCS is the same for multiple different services. I am leveraging the 835 file and the template is Wide.

Example: I have 50 charges lines that have a revenue code of 250. None of these lines have a CPT or HCPCS. This list includes 50 different drugs with 50 different prices. Meaning the expected reimbursement will vary. In this example I would be unable to appropriately identify what average dollar amount was received from the third party payer for each individual drug charge line. **Should we list the 9 nines (999999999) or the average dollar amount for all the 250 RC charge lines?**

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

At 45 CFR § 180.20 we defined “estimated allowed amount” as the average dollar amount that the hospital has historically received from a third-party payer for an item or service. Beginning on January 1, 2025, hospitals are required to encode a dollar value for the “estimated allowed amount” data element, when **a** payer-specific negotiated charge can only be expressed as an algorithm or percentage. 

Because the estimated allowed amount data element is meant to provide an estimate of what the algorithm produces in dollars, across the group of people covered by a particular payer's plan, we clarified that such an amount should reflect the amount the hospital has historically received from the payer for **the** item or service (or service package). In the Final Rule we declined to revise the definition in such a way that it might suggest that hospitals should calculate and display the average total gross charges on the claims submitted to the payer, rather than calculating and displaying the average negotiated or allowed amount that is received by the hospital, because the total gross charges are not representative of the rate negotiated between the hospital and payer. However, nothing in the hospital price transparency regulation would preclude a hospital from voluntarily including such information in the MRF in addition to including the “estimated allowed amount.” Moreover, we believe hospitals should retain flexibility, in the interest of reducing burden, to determine the best data source for calculating the estimated allowed amount. Please refer to  the discussion beginning at [88 FR 82099](https://www.federalregister.gov/d/2023-24293/p-4985).
